Mathlib/Topology/Order/Lattice.lean

```diff
@@ -133,7 +131,7 @@ end SupInf
 
 open Finset
 
-variable {ι : Type*} {s : Finset ι} {f : ι → α → L} {g : ι → L}
+variable {α ι : Type*} {s : Finset ι} {f : ι → α → L} {g : ι → L} {l : Filter α}
```

Comment:
Arguably
```suggestion
variable {ι α : Type*} {s : Finset ι} {f : ι → α → L} {l : Filter α} {g : ι → L}
```
is a better order since it matches the order things appear in the lemma